DXRFixes/DeusEx/Classes/ElevatorMover.uc

```diff
@@ -26,14 +46,19 @@ function SetSeq(int seqnum)
     if( MoveTime/2 < Level.TimeSeconds-lastTime )
         oldSeq = true;
 
+    dist = VSize(BasePos + KeyPos[seqnum] - Location);
+
+    if (KeyPos[0] == vect(0.0, 0.0, 0.0)) {
+        MoveTime = dist / GetMoveSpeed();
+    }
+
```

Comment:
Is there a reason this block of code needs to run before an early return?  Couldn't it go into the "if (KeyNum != seqnum || oldSeq)" section below, so it doesn't need to even run as frequently?

Reply:
Also, is the check for vect(0,0,0) needed?

Reply:
You're right, it should go after.